test/unit/test_default_inference_handler.py

```diff
@@ -24,7 +23,7 @@
 from six import StringIO, BytesIO
 from torch.autograd import Variable
 
-from sagemaker_pytorch_serving_container import default_inference_handler
+from sagemaker_pytorch_serving_container import default_pytorch_inference_handler
```

Comment:
the renaming seems a bit unnecessary and redundant - it is already inside sagemaker_pytorch_serving_container

Reply:
We have a module coming from sagemaker_inference as default_inference_handler. Hence the disambiguation.